clients/source_terminate_windows.go

```diff
@@ -0,0 +1,20 @@
+//go:build windows
+
+package clients
+
+func (c *SourceClient) terminateProcess() error {
+	if err := c.cmd.Process.Kill(); err != nil {
+		c.logger.Error().Err(err).Msg("failed to kill source plugin")
+	}
+	st, err := c.cmd.Process.Wait()
+	if err != nil {
+		return err
+	}
+	if !st.Success() {
+		// on windows there is no way to shutdown gracefully via signal. Maybe we can do it via grpc api?
+		// though it is a bit strange to expose api to shutdown a server :thinking?:
```

Comment:
This sounds like a reasonable way to do it, in the absence of OS support. But let's kick that can down the road for another day